engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -223,10 +224,10 @@ class WorkflowManagerActor(config: Config, val workflowStore: ActorRef)
     * Submit the workflow and return an updated copy of the state data reflecting the addition of a
     * Workflow ID -> WorkflowActorRef entry.
     */
-  private def submitWorkflow(workflow: WorkflowStore.WorkflowToStart): WorkflowIdToActorRef = {
+  private def submitWorkflow(workflow: WorkflowToStart): WorkflowIdToActorRef = {
```

Comment:
Personally I prefer the explicitness (in this case `WorkflowStore.`) when referencing foreign message classes, as sometimes the names overlap